src/query/plan/operator.cpp

```diff
@@ -4621,12 +4662,20 @@ class CallProcedureCursor : public Cursor {
   }
 
   void Reset() override {
-    result_.rows.clear();
-    result_.error_msg.reset();
-    input_cursor_->Reset();
+    self_->monotonic_memory.Release();
+    result_ =
```

Comment:
same as before, is it enough to release the memory instead of result_ destruction?